ui/src/Components/Filters/Filters.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
+import { useFilterContext } from '../../Context/filter_context';
+
+export default function Filters() {
+  const {
+    filterByEnabled,
+    filters: { status },
+    updateFilters,
+  } = useFilterContext();
+
+  const handleValueChange = (event) => {
+    const { name, value } = event.target;
+    updateFilters(name, value);
+  };
+
+  return (
+    <div className='mb-2 xl:w-96'>
```

Comment:
use `section` tag instead of `div` for:

1. consistency
2. we use section for the section partition

Reply:
renamed as per suggestion